### DISCUSSION TITLE
Remove inline CSS after a transition has ended

### DISCUSSION THREAD

**Discussion by nachtfunke** [pmndrs/react-spring#1339] [Support]
Hello!

As far as I understand the upcoming version of react-spring comes now with `onRest` for useTransition. But until it is released, I was wondering if there is some way of maybe hooking into that specific event in the current stable version?

Context: I have an Overlay that uses a transform to animate. Inside that overlay there should be a fixed element, but due to a long existing bug, `position: fixed` inside containers that are transformed [will not work](https://bugs.chromium.org/p/chromium/issues/detail?id=20574). There are some simple ways around this, for example to remove the inline style after the transition has ended. Usually I do this with the `transitionEnd` event, but that's not working in this context. Is there maybe some way to achieve this in the current version of react-spring?9

I'm thankful for every idea.

TLDR; I was wondering if there is a way to remove the inline style from a transitioned element after the transition has ended, because otherwise I won't be able to use `position: fixed`
